src/index.js

```diff
@@ -0,0 +1,5 @@
+import './index.css';
+import { Application } from '../modules/application/application.js';
```

Comment:
- [ ] As I mentioned earlier, even though you are trying to import your `Application` class to your `index.js` file, first you need to export it, so please check that. If you decide to use `export default Application`, then you don't need the brackets in this line and you can just do `import Application from '/path'`

Reply:
Done. Thanks for suggestion.